src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -2178,7 +2178,6 @@ def forward(
 
         emb = emb if aug_emb is None else emb + aug_emb
         emb = emb.repeat_interleave(repeats=num_frames, dim=0)
-        encoder_hidden_states = encoder_hidden_states.repeat_interleave(repeats=num_frames, dim=0)
```

Comment:
Yes, any existing implementation using this model from diffusers will fail after next release if they're using it outside of our supported AnimateDiff pipelines. Even our existing AnimateDiff community pipelines would fail.

Should I add a deprecation warning message here? I'm a bit unsure on how we would detect if this line should be executed or not... I'm thinking having a check that looks like:

```python
if encoder_hidden_states.shape[0] % num_frames == 0:
    # We are using the new implementation where the prompt embeds are repeated within
    # the pipeline, so we don't do anything here
    pass
else:
    deprecate(...)
    encoder_hidden_states = encoder_hidden_states.repeat_interleave(repeats=num_frames, dim=0)
```

It seems a bit unclean, or possibly incorrect, though so LMK if you know what would be best to do here...